npm/test/upload-form.test.ts

```diff
@@ -40,53 +38,57 @@ class MockDom {
+                                        <svg class="green-tick-mark" role="presentation" focusable="false" viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg" width="25px" height="25px">
+                                            <path d="M13.6,15.4h-2.3v-4.5h2.3V15.4z M13.6,19.8h-2.3v-2.2h2.3V19.8z M0,23.2h25L12.5,2L0,23.2z"/>
+                                        </svg>
                                     </span>
-                                </summary>
-                                <div class="govuk-details__text">
-                                    @Messages("upload.fileExtensionSummary")
-                                </div>
-                            </details>
-                            <button class="govuk-button" type="submit" data-module="govuk-button" role="button">
-                                @Messages("upload.continueLink")
-                            </button>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </form>`)
+                                    <p id="failureMessageText">You can only drop a single folder</p>
+                                  </dd>
+                              </div>
+                          </div>
+                          <div>
+                              <div class="govuk-form-group">
+                                  <div class="drag-and-drop__dropzone">
+                                      <input type="file" id="file-selection" name="files"
+                                          class="govuk-file-upload drag-and-drop__input" webkitdirectory
+                                          accept="image/jpeg" aria-hidden="true"
+                                      >
+                                      <p class="govuk-body drag-and-drop__hint-text">Drag and drop a single folder here or</p>
+                                      <label for="file-selection" class="govuk-button govuk-button--secondary drag-and-drop__button">
+                                      Choose folder
+                                      </label>
+                                  </div>
+                                  <p class="govuk-body">For more information on what metadata will be captured during the upload please visit our FAQ’s page</p>
+                                  <button class="govuk-button" type="submit" data-module="govuk-button" role="button">
+                                      Start upload
+                                  </button>
+                              </div>
+                          </div>
+                      </div>
+                  </div>
+              </form>
+                      <!--        Form to redirect user once upload has completed. It sends consignmentId to record processing placeholder page -->
+              @form(routes.FileChecksController.recordProcessingPage(consignmentId), Symbol("id") -> "upload-data-form") { }
```

Comment:
Is this `@form` part of the Scala templates? Is it necessary for the test? If so, could you replace with HTML? If not, we could skip it altogether.

Reply:
No, I don't think it'll be necessary; I'll remove it. Thanks.